superset/assets/src/explore/visTypes.jsx

```diff
@@ -179,7 +179,7 @@ export const visTypes = {
         expanded: true,
         controlSetRows: [
           ['color_scheme'],
-          ['show_brush', 'show_legend'],
+          ['show_brush', 'send_time_range', 'show_legend'],
```

Comment:
I know we rarely use 3 elements in the same row, but I think it works in this case:

<img width="534" alt="screen shot 2018-08-17 at 2 51 12 pm" src="https://user-images.githubusercontent.com/1534870/44290471-b34a0200-a22d-11e8-90ea-cc0c26e93d62.png">

Reply:
Checkboxes tend to look bad when combined to other controls, I'm not sure how to fix that.

Reply:
Can you convert `Show Range Filter` into another checkbox?

Reply:
@kristw `Show Range Filter` has `yes|no|auto` options ...

Reply:
Ah. I see. Thanks @mistercrunch 
I would recommend having both checkboxes on another line then.